shared/constants/chat2/index.tsx

```diff
@@ -510,6 +510,30 @@ export const getParticipantInfo = (
   return participantInfo ? participantInfo : noParticipantInfo
 }
 
+export const getConversationNameForInput = (
```

Comment:
lets not put helpers in here that only apply to one place. just move it into the input file